pkg/apis/core/v1/zz_generated.conversion.go

```diff
@@ -4836,7 +4836,7 @@ func autoConvert_v1_NodeSpec_To_core_NodeSpec(in *v1.NodeSpec, out *core.NodeSpe
 	out.Unschedulable = in.Unschedulable
 	out.Taints = *(*[]core.Taint)(unsafe.Pointer(&in.Taints))
 	out.ConfigSource = (*core.NodeConfigSource)(unsafe.Pointer(in.ConfigSource))
-	out.DoNotUse_ExternalID = in.DoNotUse_ExternalID
+	// WARNING: in.DoNotUse_ExternalID requires manual conversion: does not exist in peer-type
```

Comment:
move this to the separate PR for discussion

Reply:
I have reverted this commit.